core-bundle/src/Resources/config/migrations.yml

```diff
@@ -83,3 +83,7 @@ services:
     Contao\CoreBundle\Migration\Version412\MergeGuestsOnlyMigration:
         arguments:
             - '@database_connection'
+
+    Contao\CoreBundle\Migration\Version412\PageLanguageMigration:
```

Comment:
Did you forget to commit this class? I can not find it in this PR.

Reply:
Leftover from b5a9f84a491d6100922240c2d0f6e4b341652d8b

Reply:
Don’t we need a migration for `tl_page.language`?